### ISSUE TITLE
spell error in normalizing_flows_pytorch.py file

### ISSUE THREAD

**Issue by GreatRSingh** [deepchem/deepchem#3534]
## 🐛 Bug

deepchem\models\torch_models\normalizing_flows_pytorch.py has spelling errors.

## To Reproduce

1. Error Location -> Line no. 100 

`       Returns`
        `-------`
        `sample: tuple`
            `This tuple contains a two torch.Tensor objects. The first represents`
            `a sampling of the learned distribution when transformations had been`
            `applied. The secong torc.Tensor is the computation of log probabilities`
            `of the transformed distribution.`
            `shape: ((samples, dim), (samples))`

## Environment

* OS: Windows
* Python version: 3.8
* DeepChem version: 
* RDKit version (optional):
* TensorFlow version (optional):
* PyTorch version (optional):
* Any other relevant information:

**Comment by Kiwi-520** [comment]
hi @sudo-rsingh, 
I checked the current main branch and couldn’t find the reported typos (secong, torc.Tensor) in normalizing_flows_pytorch.py.
It looks like this may have already been fixed. Please let me know if I’m missing anything.
